### PR TITLE
Nicolas sastoque

### PR DESCRIPTION
**Pull request by NicolasZon**

### CODE REVIEW

File: Week1/NicolasSastoque/functional/src/main/java/StreamsModel/List.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+public class List {
+
+    public static void main(String[] args) {
+        System.out.println("Streams List --->");
+
+        Office office = new Office();
+
+        Table southAmericaTable = new Table();
+        ArrayList<Element> southAmericaElement = new ArrayList<>(
```
Comment (estigma88) [inline Week1/NicolasSastoque/functional/src/main/java/StreamsModel/List.java:19]: This is redundant, Arrays.asList gives you the list already created

**Review by estigma88 — CHANGES_REQUESTED** [review]
Excellent job, only a little comment there

File: Week1/NicolasSastoque/unit-test/src/test/java/kata/business/AccountServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        }
+        catch (Exception e){
+            System.out.println(e.getMessage());
+        }
+
+        Assert.assertEquals(500, currentAccount.getBalance());
+        Assert.assertEquals(4000, savingAccount.getBalance());
+    }
+
+    @Test
```
Comment (chejunior) [inline Week1/NicolasSastoque/unit-test/src/test/java/kata/business/AccountServiceImplTest.java]: Is better to use here something like this:

@Test(expected=AccountException.class)

so later you need to remove the try-catch clause, and also the asserts ones, this should be enough to pass in an easier way the fail test 👍

Comment (chejunior) [inline Week1/NicolasSastoque/unit-test/src/test/java/kata/business/AccountServiceImplTest.java:61]: Is better to use here something like this:

@test(expected=AccountException.class)

so later you need to remove the try-catch clause, and also the asserts ones, this should be enough to pass in an easier way the fail test 👍

**Review by chejunior — CHANGES_REQUESTED** [review]

**Pull request merged**